root_VS2012/programs/C#/Frameworks/Infrastructure/Public/Db/DamSqlSvr.cs

```diff
@@ -114,6 +114,11 @@ public SqlConnection DamSqlConnection
                 // コネクションを戻す
                 return _cnn;
             }
+            set
+            {
+                //set the connection
+                _cnn = value;
+            }
```

Comment:
@yusukemaegawa 

>How do you think about the evil of making a connection object modifiable?
 
I think that is the trade-off of the standardization and flexibility.